### PR TITLE
fix(element-templates): re-enable `entriesVisible`

### PR DESCRIPTION
**Pull request by philippfromme**
__Which issue does this PR address?__

Related to camunda/camunda-modeler#1975

__Acceptance Criteria__

* [x] Corresponds to the concept 
* [x] Corresponds to the design 

__Definition of Done__

* [x] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [x] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [x] passes Continuous Integration checks
* [x] available as feature branch on GitHub
  * [x] contains the cleaned up commit history
  * [x] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [x] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: lib/provider/camunda/element-templates/CustomElementsPropertiesActivator.js [hunk 1]
```diff
@@ -43,27 +48,48 @@ module.exports = CustomElementsPropertiesActivator;
 
 
 
-// helpers //////////
+// helpers ////////////////////////////////////
```
Comment (barmac) [inline lib/provider/camunda/element-templates/CustomElementsPropertiesActivator.js:51]:
```suggestion
// helpers ////////////////////////////////////////////////////////////////////////
```

**Review by nikku — APPROVED** [review]

File: lib/provider/camunda/element-templates/CustomElementsPropertiesActivator.js [hunk 2]
```diff
@@ -5,14 +5,16 @@ var inherits = require('inherits');
 var getTemplate = require('./Helper').getTemplate,
     getTemplateId = require('./Helper').getTemplateId;
 
+var isBoolean = require('lodash/isBoolean');
```
Comment (nikku) [inline lib/provider/camunda/element-templates/CustomElementsPropertiesActivator.js:8]: :heart:

File: lib/provider/camunda/element-templates/CustomElementsPropertiesActivator.js (hunk 1 above)
Comment (philippfromme) [reply lib/provider/camunda/element-templates/CustomElementsPropertiesActivator.js]: No.

Comment (barmac) [reply lib/provider/camunda/element-templates/CustomElementsPropertiesActivator.js]: Please do not necrobump old pull requests. Open a new pull request, and link the old one instead.

Comment (nikku) [reply lib/provider/camunda/element-templates/CustomElementsPropertiesActivator.js]: :walking:

**Pull request merged**